docs/SERVICE-CONSOLIDATION.md

```
@@ -0,0 +1,436 @@
+│  │ Splitter │    │ Video    │  │Cat │  │
+│  │ Worker   │    │ Worker   │  │    │  │
+│  └──────────┘    └──────────┘  └────┘  │
+│                                          │
+└─────────────────────────────────────────┘
+        │
+        ▼
+  MySQL + Minio
+```
+
+**Configuration:**
+```yaml
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)